data/tree/equality tester/47bd9ed4-f7b7-4872-ab54-9136ac251f4a.json

```diff
@@ -121,14 +121,14 @@
   "interface": {
     "inputs": {
       "2a9ca66c-22e3-4478-b06b-c1da6e93eeac": {
-        "type": "dictionary",
+        "type": "{string: any}",
```

Comment:
Type should be the same for all "tree" type: ``` any[] or {string: any}```
Also the port should rather be named as "tree" as we don't expect a dictionary here. 
Also change "dictionary" to "tree" in the description.